src/amo/api/index.js

```diff
@@ -74,23 +61,25 @@ export function getUserReviews({ userId, addonId } = {}) {
         log.warn('paging is not yet implemented');
       }
       return response.results;
-    })
-    .then((reviews) => {
-      if (addonId) {
-        log.info(`Filtering user ${userId} reviews by addonId ${addonId}`);
-        return reviews.filter((review) => review.addon.id === addonId);
-      }
-      return reviews;
     });
 }
 
-export function getLatestUserReview(params) {
-  return getUserReviews(params)
+export function getLatestUserReview({ user, addon } = {}) {
+  return new Promise((resolve) => {
+    if (!user || !addon) {
+      throw new Error('Both user and addon must be specified');
+    }
+    // The API will only return the latest user review for this add-on.
+    resolve(getReviews({ user, addon }));
+  })
     .then((reviews) => {
-      const latest = reviews.filter((review) => review.is_latest);
-      if (latest.length === 0) {
+      if (reviews.length === 1) {
+        return reviews[0];
+      } else if (reviews.length === 0) {
         return null;
       }
-      return latest[0];
+      throw new Error(
+        'Unexpectedly received multiple review objects: ' +
```

Comment:
Usually we'd use template strings here (and something like `oneLine` to combine them rather than string concat. I guess if we don't have an eslint rule preventing it though it's fine. I understand this is actually fewer lines/imports than using `oneLine` or `dedent` (though I like them more than string concat, especially when you're presumably casting it to a string anyway with the template string on the next line.